### PR TITLE
[blog] add in release announcement for 0.12

### PR DESCRIPTION
**Pull request by shea256**

### CODE REVIEW

File: _posts/en/posts/2016-02-10-release-0-12-0.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+## Ability to Limit Upload Traffic
+
+- Node upload traffic can be burdensome for some users, so the ability to put limitations on such traffic is a much-needed improvement.
+- Node users now have the ability to set soft limits on how much data they upload and serve to their peers.
+- Users can set a parameter that specifies how much data the node should target to be served daily.
+- It will try not to approach, rather than hit the limit, and after it hits the limit it will only serve blocks requested within the last week.
+
+_Credits: Jonas Schnelli_
+
+Crash Prevention via Memory Pool Limits
```
Comment (btcdrak) [inline _posts/en/posts/2016-02-10-release-0-12-0.md]: needs `##` prefix

**Comment by btcdrak** [comment]
v0.12 has been tagged so this post will be merged once the official announcement is made by @laanwj

File: _posts/en/posts/2016-02-18-release-0-12-0.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+version: 1
+---
+We're very excited to announce the official release of Bitcoin Core v0.12.0. A lot of hard work has gone into this release and it may just be the biggest one yet, with more significant improvements than any other before.
+
+Here are the major improvements you’ll get to benefit from if you upgrade your nodes to version 0.12:
+
+1. Up to 7x Faster Signature Validation
+2. Ability to Limit Upload Traffic
+3. Crash Prevention via Memory Pool Limits
+4. Option to Send Transactions That Can Be Fee-Boosted
```
Comment (morcos) [inline _posts/en/posts/2016-02-18-release-0-12-0.md]: You're talking about RBF right?  There isn't support for sending RBF txs, only for relaying them.

File: _posts/en/posts/2016-02-18-release-0-12-0.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+_Credits: Pieter Wuille, Greg Maxwell, and Cory Fields_
+
+## Ability to Limit Upload Traffic
+
+Node upload traffic can be burdensome for some users, so the ability to put limitations on such traffic is a much-needed improvement. Node users now have the ability to set soft limits on how much data they upload and serve to their peers. Users can set a parameter that specifies how much data the node should target to be served daily. It will try to stay below the limit, rather than hit it, and if it hits the limit it will only serve blocks requested within the last week.
+
+_Credits: Jonas Schnelli_
+
+Crash Prevention via Memory Pool Limits
```
Comment (morcos) [inline _posts/en/posts/2016-02-18-release-0-12-0.md]: Formatting

File: _posts/en/posts/2016-02-18-release-0-12-0.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+## Option to Send Transactions That Can Be Fee-Boosted
+
+Transactions often get stuck if they are have fees that are too low. This can cause problems because unspent transactions outputs (UTXOs) that were used in those transactions can be hard to spend, potentially freezing funds. Appropriate transaction fees are hard to calculate because they are highly dependent on the volume of transactions and their fees at any given time. Thus, one either typically undershoots and gets many stuck transactions or often overshoots, resulting in a massive overpayment and a steady loss of funds.
+
+A new feature called Opt-in Replace-by-Fee gives transaction senders the option to configure their transactions to be able to be replaced later by other transactions that specify larger fees. Senders can start with a low fee and see if their transaction gets picked up, then increase their fee until it gets accepted. This allows senders to both minimize the fees they pay and maximize the chance that their transactions will be included in a block.
+
+_Credits: Peter Todd and Suhas Daftuar_
+
+## Improved Rules for Transaction Relaying
```
Comment (morcos) [inline _posts/en/posts/2016-02-18-release-0-12-0.md]: I'm not sure what this is referring to exactly.  The fact that free transactions don't always get relayed is a byproduct of mempool limiting, not exactly a feature.  The limitations on chains of unconfirmed transactions is a feature, but that doesn't have anything to do with fees.

**Comment by morcos** [comment]
I think mention of the huge speed up in block template creation (and solving of the memory blowup) is worth mentioning.  Combined with libsecp256k1 and other validation speedups it makes 0.12 a must have for miners.

File: _posts/en/posts/2016-02-18-release-0-12-0.md (hunk 3 above)
Comment (btcdrak) [inline _posts/en/posts/2016-02-18-release-0-12-0.md]: Specifically requires ## prefix on the title.

**Pull request merged**